src/simply/simply_msg_commands.h

```diff
@@ -53,4 +53,6 @@ enum Command {
   CommandElementAnimate,
   CommandElementAnimateDone,
   NumCommands,
+  CommandVoiceStart,
+  CommandVoiceData,
```

Comment:
Add these before NumCommands and remove the NumCommands packet type in simply-pebble.js -- it isn't a real packet type, it's just to get the number of commands.